### PR TITLE
docs: explain the smc bmc license issue

### PR DESCRIPTION
**Pull request by flanfly**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [x] Documentation

As requested in #4836. I'm not sure what else to put there.

### CODE REVIEW

**Review by superm1 — COMMENTED** [review]
I think you need to reference this in the other places I suggested too so it gets published to fwupd.github.io in the next release

**Comment by superm1** [comment]
I think squash those together and then it looks good.

I had another thought, @hughsie how about if that URL (https://fwupd.github.io/libfwupdplugin/supermicro-license.html) goes into `UpdateError` or `UpdateMessage` for the device?

**Comment by hughsie** [comment]
I think putting a message in UpdateError would be helpful, yes.

File: docs/supermicro-license.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+support Redfish some features are only available after buying them as
+additional feature. One of those are applying BIOS and BMC firmware updates.
+
+## Details
+
+If you want to update your Supermicro board via redfish using fwupd you will
+need either the SFT-OOB-LIC or the SFT-DCMS-Single license.
+
+The license can be installed via redfish by POSTing it to
+`/redfish/v1/Managers/1/LicenseManager/ActivateLicense` or using the web
```
[the remark this answers is not in the document]
Comment (hughsie) [reply docs/supermicro-license.md:17]: Good idea.

**Comment by hughsie** [comment]
@flanfly can we get this over the finish line pls?

**Comment by flanfly** [comment]
I'll take care of it this week.

**Pull request merged**